pkg/notifications/notifications.go

```diff
@@ -102,7 +102,28 @@ type IdentityMonitorConfiguration struct {
 	EmailNotificationSMTP     *EmailNotificationInput    `yaml:"emailNotificationSMTP"`
 	EmailNotificationMailgun  *MailgunNotificationInput  `yaml:"emailNotificationMailgun"`
 	EmailNotificationSendGrid *SendGridNotificationInput `yaml:"emailNotificationSendGrid"`
-	TrustedCAs                []string                   `yaml:"trustedCAs"`
+	CARoots                   string                     `yaml:"caRoots"`
+	CAIntermediates           string                     `yaml:"caIntermediates"`
+}
+
+func validatePEMFile(pemFile string) error {
+	// Skip validation for empty file paths
+	if pemFile == "" {
+		return nil
+	}
+	if _, err := os.Stat(pemFile); err != nil {
+		return fmt.Errorf("invalid trusted CA file %s: %v", pemFile, err)
+	}
+	// Check if the file is a valid PEM file
+	caBytes, err := os.ReadFile(pemFile)
+	if err != nil {
+		return fmt.Errorf("failed to read trusted CA file %s: %v", pemFile, err)
+	}
+	block, _ := pem.Decode(caBytes)
+	if block == nil {
+		return fmt.Errorf("invalid trusted CA file %s: not a valid PEM file", pemFile)
+	}
```

Comment:
we could propagate the decoding error to our error message here

Reply:
`func Decode(data []byte) (p *Block, rest []byte) {` the second return value is not error here